src/test/java/pl/nogacz/chess/board/UndoTest.java

```diff
@@ -0,0 +1,31 @@
+package pl.nogacz.chess.board;
+
+import org.junit.Test;
+
+import org.junit.Assert;
+import java.util.ArrayList;
+import static org.junit.Assert.*;
+
+import pl.nogacz.chess.application.menu.Undo;
+
+public class UndoTest{
+
+    @Test
```

Comment:
- Bir kere undo yapabilir mi (yapıyorsa sağda görünün liste doğru şekilde bir azalıyor mu)
- İki kere undo yapabilir(yapıyorsa sağda görünün liste doğru şekilde iki azalıyor mu)
- 0 move olduğunda undo crash etmez (liste boştur ve boş kalır)

gibi test caseler düşün ve bunlarla ilgili test yaz.

Reply:
RemoveMovement metodu text alanını update ettiği için guisiz çağıramadım o yüzden sadece text alanını update etmeyeceği boş liste ile test ettim.